tests/std/tests/P0896R4_views_drop/test.cpp

```diff
@@ -16,6 +16,23 @@ using namespace std;
 
 #pragma warning(disable : 6011) // Dereferencing NULL pointer '%s'
 
+struct evil_convertible_to_difference {
+    evil_convertible_to_difference() = default;
+    evil_convertible_to_difference(const evil_convertible_to_difference&) {
+        throw(42);

```

Comment:
Style nitpick: `throw` isn't a function, so we conventionally write `throw 42;` without parentheses. Occurs below, and in `P0896R4_views_take/test.cpp`.